packages/flutter_tools/lib/src/build_system/targets/assets.dart

```diff
@@ -119,45 +117,3 @@ class CopyAssets extends Target {
     );
   }
 }
-
```

Comment:
This code is unused and won't be used, since we always need to create .flutter-plugins before running build